app/controllers/commands_controller.rb

```diff
@@ -52,8 +52,14 @@ def update
   end
 
   def destroy
-    @command.destroy
-    successful_response('Command removed.')
+    if @command.destroy
+      successful_response('Command removed.')
+    else
+      respond_to do |format|
+        format.html { render :show }
```

Comment:
I think it will be good to keep this just in case. Good for API consumers as well.